modules/andBeyondMediaBidAdapter.js

```diff
@@ -4,7 +4,7 @@ import { registerBidder } from '../src/adapters/bidderFactory.js';
 import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
 import { config } from '../src/config.js';
 
-const BIDDER_CODE = 'beyondmedia';
+const BIDDER_CODE = 'andbeyond.media';
```

Comment:
This is very long, why not andbeyond

Reply:
It's also breaking i believe

Reply:
> This is very long, why not andbeyond

The thing is that this is a full name of the Company and we cannot shorten this to andbeyond. I haven't seen any limitations before. Do you have any?

Reply:
GAM key values have a limit of 20 chars. And GET requests have a length limit. This can cause issues for publishers which use `enableSendAll` key values.

Reply:
@patmmccann @muuki88 
We have changed the code to "andbeyond". This is fine ?

Reply:
You still need the old name as an alias so as to not break existing builds

Reply:
@patmmccann @muuki88 we noticed that biddercode `andbeyond` is an Adkernel adapter alias, is that normal?https://github.com/AndBeyondMediaHB/Prebid.js/blob/master/modules/adkernelBidAdapter.js#L82

Reply:
@patmmccann @muuki88 
![image](https://user-images.githubusercontent.com/107686862/192513767-499ff194-d9e4-41c5-886d-cf97ff661524.png)

Are these rules valid? If so, can we use the biddercode `andbeyondmedia`. If not, why not?

Reply:
well in this case you would not be unique with adkernel's alias for the first 6 characters. Did you discover why they are using your name?

Reply:
@patmmccann They are using this name because this is a name of their company, they cannot change it to another one. Could you please clarify if it can cause some problems when this is not unique with adkernel's alias?

Reply:
AdKernel's company name is also andbeyond?